torchvision/models/quantization/inception.py

```diff
@@ -142,6 +134,8 @@ def __init__(
                 QuantizableInceptionE,
                 QuantizableInceptionAux,
             ],
+            *args,
```

Comment:
@pmeier Since you are the most Pythonic person I know, any idea why this works? We pass the `inception_blocks` keyword parameter before *args. We do exactly the same thing on `QuantizableGoogLeNet`. Could this because we dont actually have any positional args in the classes? I was expecting that the class would emit an error for putting positional arguments after named params but it doesn't.

Reply:
This is one of the Python quirks that are syntactically allowed, but should never be used due to its very weird behavior. TL;DR: `foo(baz="baz", "bar")` is not valid syntax while `foo(baz="baz", *["bar"])` is. See python/cpython#82741.

I would advise to put the `*args` before the `inception_blocks=...` parameter to avoid confusion. Same for all other occurrences of this pattern. I don't know of a linter that checks this for us though.

Reply:
Done! 👍 
LGTM now?

Reply:
Not sure if we want to do it in this PR or split this off to a follow-up cc @datumbox. Nevertheless this bothered me a bit to much TBH. I wrote a small `flake8` plugin to find all occurrences of this. Put the following in a `nuakw.py` file in the root of the project

```py
import ast
import contextlib


class NoUnpackingAfterKeywordFinder(ast.NodeVisitor):
    def __init__(self):
        self.errors = []

    def visit_Call(self, node: ast.Call):
        with contextlib.suppress(StopIteration):
            starred = next(starred for starred in node.args if isinstance(starred, ast.Starred))
            if any(
                (keyword.value.lineno, keyword.value.col_offset) < (starred.lineno, starred.col_offset)
                for keyword in node.keywords
            ):
                self.errors.append(
                    (node.lineno, node.col_offset, "NUAKW Argument unpacking after keyword argument", type(self))
                )

        self.generic_visit(node)


class NoUnpackingAfterKeyword:
    name = "no_unpacking_after_keyword"
    version = "0.0.1"

    def __init__(self, tree: ast.AST):
        self._tree = tree

    def run(self):
        visitor = NoUnpackingAfterKeywordFinder()
        visitor.visit(self._tree)
        yield from visitor.errors

```

Now, put the following at the bottom of `setup.cfg`

```
[flake8:local-plugins]
extension =
    NUAKW = nuakw:NoUnpackingAfterKeyword
paths =
    ./

```

Running `flake8 torchvision` yields the following errors:

```
torchvision/models/quantization/inception.py:44:9: NUAKW Argument unpacking after keyword argument
torchvision/models/quantization/inception.py:55:9: NUAKW Argument unpacking after keyword argument
torchvision/models/quantization/inception.py:66:9: NUAKW Argument unpacking after keyword argument
torchvision/models/quantization/inception.py:77:9: NUAKW Argument unpacking after keyword argument
torchvision/models/quantization/inception.py:88:9: NUAKW Argument unpacking after keyword argument
torchvision/models/quantization/inception.py:122:9: NUAKW Argument unpacking after keyword argument
torchvision/models/quantization/googlenet.py:42:9: NUAKW Argument unpacking after keyword argument
torchvision/models/quantization/googlenet.py:53:9: NUAKW Argument unpacking after keyword argument
torchvision/models/quantization/googlenet.py:77:9: NUAKW Argument unpacking after keyword argument
torchvision/models/quantization/mobilenetv3.py:86:9: NUAKW Argument unpacking after keyword argument
```

If we fix one, we should fix all.

Reply:
This seemed to be a short fix, so I have implement this in this PR itself :)

Reply:
cc @pmeier @datumbox Shall we go to master now?